src/components/ConsoleInput.jsx

```diff
@@ -14,12 +15,33 @@ export default class ConsoleInput extends Component {
     bindAll(this, '_ref');
   }
 
-  componentWillReceiveProps({isTextSizeLarge}) {
-    if (isTextSizeLarge !== this.props.isTextSizeLarge) {
+  componentDidMount() {
+    this._focusRequestedLine(this.props.requestedFocusedLine);
```

Comment:
Seems more strictly correct to do this in the `_ref()` method, if there is an editor element?